fuzz/fuzz.js

```diff
@@ -38,38 +38,48 @@ function formatError(num, error) {
   );
 }
 
-const boringRegex = /^[\s;]*$|with/;
-
-let randomAST;
-let randomJS;
-do {
-  randomAST = esfuzz.generate({ maxDepth: 7 });
-  randomJS = esfuzz.render(randomAST);
-} while (boringRegex.test(randomJS));
-
-const options = randomOptions();
-
-let prettierJS1 = null;
-let prettierJS1Error = null;
-try {
-  prettierJS1 = prettier.format(randomJS, options);
-} catch (error) {
-  prettierJS1Error = error;
-}
+const boringRegex = /^[\s;]*$|with|\b0|:;/;
+
+let tryCount = 0;
+while (true) {
+  tryCount++;
+  var randomAST = esfuzz.generate({ maxDepth: 7 });
```

Comment:
I changed all of those from let/const to var so that they can be used outside of the loop instead of defining them all at the top.